test/acceptance/expected-output/key-event.js

```diff
@@ -0,0 +1,12 @@
+import { keyEvent, visit } from 'ember-native-dom-helpers';
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('keyEvent');
+
+test('visiting /foo', async function(assert) {
+  await visit('/foo');
+
+  await keyEvent('#bar', 'keypress', 13);
+  assert.equal(currentURL(), '/foo');
```

Comment:
This is something I might want to check with @rwjblue, but this addon also has `currentURL`, `currentRouteName` and that (they are really just shims to `window.XXX`). One of the points is avoid globals.
Perhaps we should also transform those. If we do that, we don't need the special ```
  env: {
    embertest: true
  },
``` in the `.eslintrc.js` file. 

Don't do it yet, but it would be a simple transform. Just detect when the function is called and add the import statement.